libs/dyn/value.go

```diff
@@ -103,7 +121,7 @@ func (v Value) AsAny() any {
 	case KindMap:
 		m := v.v.(Mapping)
 		out := make(map[string]any, m.Len())
-		for _, pair := range m.pairs {
+		for _, pair := range m.Pairs() {
```

Comment:
Because ordering on the output doesn't matter here, you can probably iterate over `.data` directly.